### PR TITLE
add compare page

### LINKED ISSUE #76: compare package version between distros
Similar to http://www.ros.org/debbuild/compare_hydro_indigo.html on the old build farm.

### PR DESCRIPTION
**Pull request by dirk-thomas**
Fixes #76

http://repositories.ros.org/status_page/compare_indigo_jade.html

### CODE REVIEW

File: ros_buildfarm/templates/status/release_compare_page_job.xml.em [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        'python3 -u $WORKSPACE/ros_buildfarm/scripts/subprocess_reaper.py $$ --cid-file $WORKSPACE/docker_generate_compare_page/docker.cid > $WORKSPACE/docker_generate_compare_page/subprocess_reaper.log 2>&1 &',
+        '# sleep to give python time to startup',
+        'sleep 1',
+        '',
+        '# generate Dockerfile, build and run it',
+        '# generating the release compare page',
+        'echo "# BEGIN SECTION: Generate Dockerfile - compare page"',
+        'export TZ="%s"' % timezone,
+        'export PYTHONPATH=$WORKSPACE/ros_buildfarm:$PYTHONPATH',
+        'python3 -u $WORKSPACE/ros_buildfarm/scripts/status/run_release_compare_page_job.py' +
```
Comment (tfoote) [inline ros_buildfarm/templates/status/release_compare_page_job.xml.em]: Is this naming consistent with our other projects. The script being called run_* where it actually is a Docker file generator does not make sense to me reading this.

Comment (dirk-thomas) [inline ros_buildfarm/templates/status/release_compare_page_job.xml.em]: This is exactly the same naming theme as for the status pages. And as far as I can see even all the other scripts invoking Docker are called `run_..._job`.

Comment (tfoote) [inline ros_buildfarm/templates/status/release_compare_page_job.xml.em]: ok, at least it's consistent

**Comment by tfoote** [comment]
+1

**Pull request merged**